src/test/Test.FAKECore/SemVerHelperSpecs.cs

```diff
@@ -96,9 +96,9 @@ public class when_comparing_semvers
             () => SemVerHelper.parse("1.0.0-alpha.1")
                 .ShouldBeLessThan(SemVerHelper.parse("1.0.0-alpha.beta"));
 
-        It should_assume_alpha_is_smaller_tha_beta =
+        It should_assume_that_longer_prereleases_are_greater =
```

Comment:
I kept the test data, but per the spec for these inputs 1.0.0-beta is larger, because a shorter prerelease tag is bigger than a longer prerelease tag.